pkg/services/ngalert/api/tooling/definitions/provisioning_contactpoints.go

```diff
@@ -152,7 +152,7 @@ func (e *EmbeddedContactPoint) SecretKeys() ([]string, error) {
 	case "victorops":
 		return []string{}, nil
 	case "webhook":
-		return []string{}, nil
+		return []string{"password", "authorization_credentials"}, nil
```

Comment:
`password` seems to have slipped from other PRs and it was caught by the new test introduced.